test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -602,6 +602,21 @@ public void HandleTypeDefsUseIntPtrAsFieldType(string handleType)
         Assert.Equal(nameof(IntPtr), Assert.IsType<IdentifierNameSyntax>(field.Declaration.Type).Identifier.ValueText);
     }
 
+    [Theory]
+    [InlineData("POINT")]
+    [InlineData("POINTF")]
+    [InlineData("RECT")]
+    [InlineData("SIZE")]
+    public void DrawingStructsGenerate(string handleType)
+    {
+        this.compilation = this.compilation.WithOptions(this.compilation.Options.WithPlatform(Platform.X64));
+        this.generator = this.CreateGenerator();
+        Assert.True(this.generator.TryGenerate(handleType, CancellationToken.None));
+        this.CollectGeneratedCode(this.generator);
+        this.AssertNoDiagnostics();
+        StructDeclarationSyntax paintStruct = Assert.IsType<StructDeclarationSyntax>(this.FindGeneratedType(handleType).Single());
+    }
```

Comment:
Thanks for the test, but we already have exhaustive tests to make sure nothing generated will produce a compiler warning or error. And testing that the type is generated when requested by name isn't something we do more than once or twice total, rather than for every type we customize. So please drop the test.

A more meaningful test would be in `BasicTests.cs`. There you can *use* the APIs you're adding via the template to confirm they are present and that they have correct implementations.